doc/convolution.md

```diff
@@ -263,6 +263,59 @@ Outputs something like:
 [torch.DoubleTensor of dimension 2x4x3]
 ```
 
+LookupTable supports max-norm regularization. One can activate the max-norm constraints
```

Comment:
You should also add the new parameters above to the line:

`module = nn.LookupTable(nIndex, size, [paddingValue])`